test/e2e/validators.go

```diff
@@ -50,6 +49,24 @@ func cleanupOptions(ctx *framework.TestCtx) *framework.CleanupOptions {
 	}
 }
 
+// clientGet is a wrapper around f.Client.Get that performs retries in case of retryable errors
+func clientGet(key types.NamespacedName, obj runtime.Object) error {
+	f := framework.Global
+
+	return wait.PollImmediate(1*time.Second, 60*time.Second, func() (bool, error) {
```

Comment:
I find the 1 second interval to aggressive, can we increase to 4 or 5 seconds?

Reply:
@qu1queee changed to fours seconds.